compliance/github/empty_repository/empty_repository.pt

```diff
@@ -0,0 +1,137 @@
+name "GitHub.com Empty Repository Report"
+rs_pt_ver 20180301
+short_description "Gets the repositories under a GitHub.com Organization and creates incidents for any that do have any commits within a certain number of days.  See the [README](https://github.com/rightscale/policy_templates/tree/master/compliance/github/empty_repository) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+severity "low"
+category "Compliance"
+
+permission "read_creds" do
+  actions   "rs_cm.show_sensitive","rs_cm.index_sensitive"
+  resources "rs_cm.credentials"
+end
+
+parameter "param_email" do
+  label "Email addresses of the recipients you wish to notify"
+  type "list"
+end
+
+parameter "param_orgs" do
+  type "list"
+  label "Organizations on GitHub.com to scan"
+end
+
+parameter "param_repo_age" do
+  type "string"
+  label "Required minimum age of repository in bytes"
```

Comment:
age in bytes?

Reply:
Fixed in commit 6e30a8af8b92d2b18b4edfd835e5f1d3245117fb